src/Components/ServiceScene/LogsPanelScene.tsx

```diff
@@ -280,42 +280,37 @@ export class LogsPanelScene extends SceneObjectBase<LogsPanelSceneState> {
-        .setOption('onClickFilterOutLabel', this.handleLabelFilterOutClick)
-        .setOption('isFilterLabelActive', this.handleIsFilterLabelActive)
-        .setOption('onClickFilterString', this.handleFilterStringClick)
-        .setOption('onClickFilterOutString', this.handleFilterOutStringClick)
-        .setOption('onClickShowField', this.onClickShowField)
-        .setOption('onClickHideField', this.onClickHideField)
-        .setOption('displayedFields', parentModel.state.displayedFields)
-        .setOption('sortOrder', options.sortOrder ?? getLogsPanelSortOrderFromStore())
-        .setOption('wrapLogMessage', options.wrapLogMessage ?? Boolean(getLogOption<boolean>('wrapLogMessage', false)))
-        .setOption(
-          'prettifyLogMessage',
-          options.prettifyLogMessage ?? Boolean(getLogOption<boolean>('wrapLogMessage', false))
-        )
-        .setMenu(
-          new PanelMenu({
-            investigationOptions: { type: 'logs', getLabelName: () => `Logs: ${getPrettyQueryExpr(serviceScene)}` },
-          })
-        )
-        .setOption('showLogContextToggle', true)
-        // @ts-expect-error Requires Grafana 11.5
-        .setOption('enableInfiniteScrolling', true)
-        // @ts-expect-error Grafana 11.5
-        .setOption('onNewLogsReceived', this.updateVisibleRange)
-        // @ts-expect-error Grafana 11.5
-        .setOption('logRowMenuIconsAfter', [<CopyLinkButton onClick={this.handleShareLogLineClick} key={0} />])
-
-        .setHeaderActions(
-          new LogOptionsScene({ visualizationType, onChangeVisualizationType: parentModel.setVisualizationType })
-        )
-        .build()
-    );
+    return PanelBuilders.logs()
+      .setTitle(this.getTitle(serviceScene.state.logsCount))
+      .setOption('showTime', true)
+      .setOption('onClickFilterLabel', this.handleLabelFilterClick)
+      .setOption('onClickFilterOutLabel', this.handleLabelFilterOutClick)
+      .setOption('isFilterLabelActive', this.handleIsFilterLabelActive)
+      .setOption('onClickFilterString', this.handleFilterStringClick)
+      .setOption('onClickFilterOutString', this.handleFilterOutStringClick)
+      .setOption('onClickShowField', this.onClickShowField)
+      .setOption('onClickHideField', this.onClickHideField)
+      .setOption('displayedFields', parentModel.state.displayedFields)
+      .setOption('sortOrder', options.sortOrder ?? getLogsPanelSortOrderFromStore())
+      .setOption('wrapLogMessage', options.wrapLogMessage ?? Boolean(getLogOption<boolean>('wrapLogMessage', false)))
+      .setOption(
+        'prettifyLogMessage',
+        options.prettifyLogMessage ?? Boolean(getLogOption<boolean>('wrapLogMessage', false))
+      )
+      .setMenu(
+        new PanelMenu({
+          investigationOptions: { type: 'logs', getLabelName: () => `Logs: ${getPrettyQueryExpr(serviceScene)}` },
+        })
+      )
+      .setOption('showLogContextToggle', true)
```

Comment:
Looks like the logs panel is still only showing the menu icon on hover
```suggestion
      .setOption('showLogContextToggle', true)
      .setShowMenuAlways(true)
```